packages/system/external-dns/Makefile

```diff
@@ -0,0 +1,10 @@
+export NAME=external-dns
+export NAMESPACE=cozy-$(NAME)
+
+include ../../../scripts/package-system.mk
+
+update:
+	rm -rf charts
+	helm repo add bitnami https://charts.bitnami.com/bitnami
+	helm repo update bitnami
+	helm pull bitnami/external-dns --untar --untardir charts
```

Comment:
Bitnami has very specific helm charts, have you considered using the official one?

https://github.com/kubernetes-sigs/external-dns/tree/master/charts/external-dns